src/components/WalletModal/Option.tsx

```diff
@@ -183,21 +180,21 @@ export default function Option({ connection }: OptionProps) {
 
   const walletConnectV2AsDefault = useWalletConnectV2AsDefault()
 
-  const handleClickConnectViaWCv1 = (e: MouseEvent<HTMLButtonElement>) => {
+  const handleClickConnectViaPopover = (e: MouseEvent<HTMLButtonElement>) => {
     e.stopPropagation()
-    tryActivation(walletConnectV2AsDefault ? walletConnectV1Connection : walletConnectV2Connection, () => {
-      setWC1PromptOpen(false)
+    tryActivation(walletConnectV2Connection, () => {
+      setPopoverOpen(false)
       toggleAccountDrawerOpen()
     })
   }
-  const handleClickOpenWCv1Tooltip = (e: MouseEvent<HTMLButtonElement>) => {
+  const handleClickOpenPopover = (e: MouseEvent<HTMLButtonElement>) => {
     e.stopPropagation()
-    setWC1PromptOpen(true)
+    setPopoverOpen(true)
   }
 
   const isWalletConnect =
     connection.type === ConnectionType.WALLET_CONNECT || connection.type === ConnectionType.WALLET_CONNECT_V2
-  const showExtraMenuToggle = isWalletConnect && !isCurrentOptionPending
+  const showExtraMenuToggle = isWalletConnect && !isCurrentOptionPending && !walletConnectV2AsDefault
```

Comment:
This is the crux of it, right? If `!walletConnectV2AsDefault` then we shouldn't show V1 as an option, even?

Reply:
isn't that the case?